scripts/blinddeathsaves.js

```diff
@@ -9,4 +9,53 @@ Hooks.on("preCreateChatMessage", (msg, options, userId) => {
     };
     msg.data.update(updates);
   }
-});
\ No newline at end of file
+});
+
+Hooks.on("init", () => {
+  // Register settings to hide Death Save from player's character sheet
+  game.settings.register("blind-death-saves", "hiddenDeathSaveStatus", {
+    name: "Hide Death Save Status",
+    hint: "Death Save status counters will be hidden for Players in their character sheets.",
+    type: Boolean,
+    default: false,
+    scope: "world",
+    config: true,
+    restricted: true,
+  });
+});
+
+// Remove death save counters from character sheet (only for Players)
+Hooks.on("renderActorSheet", async function (app, html, data) {
+  if (
+    game.settings.get("blind-death-saves", "hiddenDeathSaveStatus") &&
+    !game.user.isGM
+  ) {
+    //Tidy5e counters
+    const tidy5eModule = game.modules?.get("tidy5e-sheet")?.active;
```

Comment:
This makes it so it doesn't work with the default Sheet if tidy5e is activated. Should maybe also check which sheet is selected by the actor/player.

Reply:
Pushed an updated logic to check for sheet class instead of active module.